castai/policies/iam-policy.json

```diff
@@ -40,7 +40,8 @@
         "ec2:CreateSecurityGroup",
         "ec2:CreateKeyPair",
         "ec2:DeleteKeyPair",
-        "ec2:CreateTags"
+        "ec2:CreateTags",
+        "ec2:ImportKeyPair"
```

Comment:
why is this needed?

Reply:
For feature use to import key pairs to aws.